tool_collections/fastx_toolkit/fastx_barcode_splitter/fastx_barcode_splitter.xml

```diff
@@ -1,10 +1,14 @@
-<tool id="cshl_fastx_barcode_splitter" version="1.0.0" name="Barcode Splitter">
+<tool id="cshl_fastx_barcode_splitter" version="@VERSION@" name="Barcode Splitter">
```

Comment:
The PR has been opened from the `test_fastx` branch, if you copy over the tool files here and do a PR we can get it in this PR.